### PR TITLE
[1.4] Fix dockerbuild 

### LINKED ISSUE #1362: release-1.4使用make docker-build构建镜像出错
release-1.4使用make docker-build构建镜像出错，信息如下：

> => ERROR [stage-1 2/5] RUN apk add --no-cache ca-certificates=\~20220614-r4 bash=\~5.2.15-r0 expat=\~2.5.0-r0   &&  4.3s
\------
 \> [stage-1 2/5] RUN apk add --no-cache ca-certificates=\~20220614-r4 bash=\~5.2.15-r0 expat=\~2.5.0-r0   && rm -rf /var/cache/apk/*:
#6 0.365 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/main/aarch64/APKINDEX.tar.gz
#6 3.250 fetch https://dl-cdn.alpinelinux.org/alpine/v3.17/community/aarch64/APKINDEX.tar.gz
#6 4.227 ERROR: unable to select packages:
#6 4.243   ca-certificates-20230506-r0:
#6 4.243     breaks: world[ca-certificates~20220614-r4]
\------
executor failed running [/bin/sh -c apk add --no-cache ca-certificates=~20220614-r4 bash=~5.2.15-r0 expat=~2.5.0-r0   && rm -rf /var/cache/apk/*]: exit code: 1
make: *** [docker-build] Error 1

怎么调整能使其正常构建？

### PR DESCRIPTION
**Pull request by liangyuanpeng**
### Ⅰ. Describe what this PR does

- Add workflow of docker build image
- Fix docker build

Fixes #1362

### CODE REVIEW

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -32,6 +32,27 @@ jobs:
         with:
           config: ./typos.toml
 
+  docker-build:
```
Comment (liangyuanpeng) [inline .github/workflows/ci.yaml:35]: Add CI to run docker build, does make sense? i can update it to master if necessarily.

File: .github/workflows/docker-image.yaml [hunk 2]
```diff
@@ -0,0 +1,15 @@
+name: Docker Image CI
```
Comment (liangyuanpeng) [inline .github/workflows/docker-image.yaml:1]: backport PR https://github.com/openkruise/kruise/pull/1393 to 1.4

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -32,6 +32,27 @@ jobs:
...
+    runs-on: ubuntu-18.04
```
Comment (liangyuanpeng) [inline .github/workflows/ci.yaml:36]: Maybe we need to update the os version? it's out-of-date.

The ubuntu-18.04 environment is deprecated, consider switching to ubuntu-20.04(ubuntu-latest), or ubuntu-22.04 instead. For more details see https://github.com/actions/virtual-environments/issues/6002

**Comment by liangyuanpeng** [comment]
@zmberg 
PTAL,Thanks.

File: .github/workflows/docker-image.yaml [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Build the Docker image
+        run: IMG=openkruise/kruise-manager:${{ github.ref_name }} & make docker-multiarch
```
Comment (zmberg) [inline .github/workflows/docker-image.yaml:15]: make docker-multiarch will automatically push the image to the repository, this do not want, but you can play the image locally to see if it is successful, but do not push the repository

**Pull request closed without merging**